content/en/blog/2024/kubecon-china-2024.md

```diff
@@ -0,0 +1,56 @@
+---
+title:
+  OpenTelemetry Talks at Kubecon + CloudNativeCon + Open Source Summit + AI_dev
+  China 2024
+linkTitle: Kubecon China 2024
+date: 2024-06-25
```

Comment:
```suggestion
date: 2024-07-10
```

Let's aim to publish this ~6 weeks before the event.

Also we should create an announcement

Reply:
I can create the announcement. Do we want to link to this blog post rather than the conference website?

Reply:
yes, link to the blog post! Let me dig up previous instances of that since we changed the approach for that

Reply:
https://github.com/open-telemetry/opentelemetry.io/pull/3983/files#diff-b2270b5c8b30e80828a875326979d047fef49466a104fcba82a4e56401984485

Reply:
Added the announcement in 4c85139. Thanks!